invenio_vocabularies/contrib/awards/schema.py

```diff
@@ -90,10 +91,21 @@ def validate_data(self, data, **kwargs):
         id_ = data.get("id")
         number = data.get("number")
         title = data.get("title")
-        if not id_ and not (number and title):
-            raise ValidationError(
-                _("An existing id or number/title must be present."), "award"
-            )
+        try:
+            ALTERNATE_FUNDING_SCHEMA = current_app.config["ALTERNATE_FUNDING_SCHEMA"]
+        except KeyError:
+            ALTERNATE_FUNDING_SCHEMA = None
+        if ALTERNATE_FUNDING_SCHEMA:
+            if not id_ and not (number or title):
```

Comment:
If not required by DataCite, we might want to relax the constraint without adding extra conditions.
I also agree that a config for this specific part might not be the best solution. Let's discuss it quickly IRL.

Reply:
I was thinking if you set `ALTERNATE_FUNDING_SCHEMA` to `True` it would check for number _or_ title. Else it would check as is for number _and_ title.

Also I thought that making it configurable (I am with you that this is not the most elegant way) it wouldn't break stuff for everyone else while enabling us to change it ;)